pkg/api/routes_gallery.go

```diff
@@ -27,9 +27,16 @@ func (rs galleryRoutes) File(w http.ResponseWriter, r *http.Request) {
 	thumb := r.URL.Query().Get("thumb")
 	w.Header().Add("Cache-Control", "max-age=604800000") // 1 Week
 	if thumb == "true" {
-		_, _ = w.Write(gallery.GetThumbnail(fileIndex))
-	} else {
+		_, _ = w.Write(gallery.GetThumbnail(fileIndex, 200))
+	} else if thumb == "" {
 		_, _ = w.Write(gallery.GetImage(fileIndex))
+	} else {
+		height, err := strconv.ParseInt(thumb, 0, 64)
```

Comment:
isn't this `width` actually instead of `height` ?

Reply:
Yup, forgot to update this file. Fixed.